trajectory/Mission.py

```diff
@@ -349,11 +349,9 @@ def check_waypoint(self):
                 if distance <= self.closeness:
                     print("Close enough: skip to next waypoint")
                     self.vehicle.set_next_waypoint()
-                    next_waypoint = next_waypoint + 1
+                    next_waypoint += 1
 
-        num_commands = self.vehicle.count_waypoints()
-        if next_waypoint >= num_commands:
-            print("Exit 'standard' mission when heading for final waypoint ({})".format(num_commands))
+        if next_waypoint >= self.vehicle.count_waypoints():
```

Comment:
Can we make this a oneliner? `return next_waypoint < self.vehicle.count_waypoints()`